### DISCUSSION TITLE
此软件可以充当qb备份吗

### DISCUSSION THREAD

**Discussion by BackMountainDevil** [ludoux/qbittorrent-torrents-exporter#3] [通用 | General]
该软件目前可以导出分类分文件夹分tracker导出种子，是否可以用来从种子中恢复或者解决强制校验问题？

**Comment by ludoux** [comment]
它仅可以导出指定的种子文件（在正常工作的情况下）。可以通过导出种子、删除任务，再以忽略哈希检验的方法添加，来忽略检验。

需要注意的是，最保险的备份任务的方法是备份 qB 的BT_backup 文件夹。☺️

**Comment by BackMountainDevil** [comment]
不得连rss也备份？

From An idea

**Reply by ludoux** [reply]
确实，把整个 qB 配置文件夹备份下来是最稳妥的

**Comment by BackMountainDevil** [comment]
靓仔，这个软件导出种子很好使，就是感觉它是不是却一个配套功能--导入。

**Comment by ludoux** [comment]
可以理解需要手动导入。比如假如不在意分类和tag信息的话，可以直接按同一个路径下的所有torrent文件来一批一批的导入。

感觉备份最稳妥的还是直接备份BT_backup文件夹

________________________________
